### ISSUE TITLE
TensorFlow Lite `--int8` export failure: `EndVector() takes 1 positional argument but 2 were given`

### ISSUE THREAD

**Issue by phodgers** [ultralytics/yolov5#5707]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and found no similar bug report.

### YOLOv5 Component

Export

### Bug

Using https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb below export fails

```
!git clone https://github.com/ultralytics/yolov5  # clone
%cd yolov5
%pip install -qr requirements.txt  # install

from yolov5 import utils
display = utils.notebook_init()  # checks

!python export.py --weights yolov5s.pt --include tflite --int8
```

At the end of export, the following message is displayed

> TensorFlow Lite: export failure: EndVector() takes 1 positional argument but 2 were given

![image](https://user-images.githubusercontent.com/35833843/142435076-767c569e-8c0a-4df0-8586-979944dae96f.png)

According to tensorflow/tensorflow#52954 there is some issue with flatbuffers library. Workaround is to downgrade flatbuffers.

The following code therefore works as expected.
```
!git clone https://github.com/ultralytics/yolov5  # clone
%cd yolov5
%pip install -qr requirements.txt  # install

from yolov5 import utils
display = utils.notebook_init()  # checks

%pip install --upgrade flatbuffers==1.12 # downgrade from v2 to v1.12
!python export.py --weights yolov5s.pt --include tflite --int8
```

### Environment

Ultralytics colab environment https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb

### Minimal Reproducible Example

```
!git clone https://github.com/ultralytics/yolov5  # clone
%cd yolov5
%pip install -qr requirements.txt  # install

from yolov5 import utils
display = utils.notebook_init()  # checks

!python export.py --weights yolov5s.pt --include tflite --int8
```

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
@phodgers thanks for the bug report! It seems like a package change may have caused this because this was working recently, so I agree your flatbuffers downgrade seems like a good workaround.
```python
%pip install --upgrade flatbuffers==1.12 # downgrade from v2 to v1.12
```

**Comment by glenn-jocher** [comment]
@phodgers good news 😃! Your original issue may now be fixed ✅ in PR #6216. This PR enforces `flatbuffers==1.12` on TFLite `--int8` export.

NOTE: This will install `flatbuffers==1.12` automatically but you will need to **restart your environment for change to take effect**.

To receive this update:
- **[Git](https://github.com/ultralytics/yolov5)** – `git pull` from within your `yolov5/` directory or `git clone https://github.com/ultralytics/yolov5` again
- **[PyTorch Hub](https://pytorch.org/hub/ultralytics_yolov5/)** – Force-reload `model = torch.hub.load('ultralytics/yolov5', 'yolov5s', force_reload=True)`
- **[Notebooks](https://github.com/ultralytics/yolov5/blob/master/tutorial.ipynb)** – View updated notebooks  <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **[Docker](https://hub.docker.com/r/ultralytics/yolov5)** – `sudo docker pull ultralytics/yolov5:latest` to update your image <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>

Thank you for spotting this issue and informing us of the problem. Please let us know if this update resolves the issue for you, and feel free to inform us of any other **issues** you discover or **feature requests** that come to mind. Happy trainings with YOLOv5 🚀!
